src/subsplit_dag.hpp

```diff
@@ -454,7 +455,7 @@ class SubsplitDAG {
   size_t CreateAndInsertEdge(const size_t parent_id, const size_t child_id,
                              bool rotated);
   // Add edge between given parent and child nodes to the DAG.
-  void ConnectGivenNodes(const size_t parent_id, const size_t child_id, bool rotated);
+  void ConnectGivenNodes(const size_t parent_id, const size_t child_id, bool rotated, LineId edge_id);
```

Comment:
I wonder if the `Line` terminology could be limited to just the internals of the storage. Can we just use `EdgeId` here?

Reply:
You're correct, I'll change it